US-CBP/cbp-theme
# Fix scrollbar- Safari Compatibility Fix

#### What's this PR do?
-fix compatibility with safari 
-small spacing fix for `tables.js`
- changed tabs to keep them consistent

#### Where should the reviewer start?
- please check the code for `ds-common-styles.scss` to see if it is working properly

#### How should this be manually tested?
- visit any of the foundation pages on chrome, safari, and firefox to see if the tabs are being display correctly

#### Any background context you want to provide?
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Questions:
- Is there a blog post?
- Does the knowledge base need an update?
